### ISSUE TITLE
Using variable expansion under loader config does not work.

### ISSUE THREAD

**Issue by amuntean-godaddy** [openconfig/gnmic#545]
The variables `ARISTA_USERNAME` and `ARISTA_PASSWORD` are not replaced with their value.

```
insecure: true
encoding: json_ietf
log: true
clustering:
  cluster-name: cluster1
  targets-watch-timer: 30s
  locker:
    type: consul
    address: gmonitor-consul-server:8500
    
loader:
  type: consul
  address: gmonitor-consul-server:8500
  services:
    - name: gnmi
      config:
        insecure: true
        username: ${ARISTA_USERNAME}
        password: ${ARISTA_PASSWORD}
        subscriptions:
          - arista-dcs
      
subscriptions:
  arista-dcs:
    paths:
      - /interfaces
      - /components/component/cpu
      - /components/component/state/memory
      - /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state
    mode: stream
    stream-mode: sample
    sample-interval: 10s

outputs:
  output1:
    type: prometheus
    append-subscription-name: false
    service-registration:
      address: gmonitor-consul-server:8500 
  ```

**Comment by karimra** [comment]
Should be fixed in v0.39.0, please give it a try.

**Comment by amuntean-godaddy** [comment]
The issue has been fixed by v0.39.0.

Thank you.
